app/models/trip.rb

```diff
@@ -5,6 +5,7 @@ class Trip < ApplicationRecord
   has_many :invitations, dependent: :destroy
   has_many :trip_activities, dependent: :destroy
   has_many :activities, through: :trip_activities
+  has_one_attached :photo
```

Comment:
We might want to save photo if we offer to attach photos, might be too generic for this use. Maybe "static_map" ?